stock_package_autoload/models/stock_move.py

```diff
@@ -0,0 +1,80 @@
+import json
+
+from odoo import api, fields, models
+
+
+class StockMove(models.Model):
+    _inherit = "stock.move"
+
+    package_domain = fields.Char(
+        compute="_compute_package_domain",
+        readonly=True,
+        store=False,
+    )
+    load_products_from_package_id = fields.Many2one(
+        "stock.quant.package",
+        string="Add package contents",
+        help="Autoresets after use",
+    )
+
+    def _package_domain(self):
+        self.ensure_one()
+        to_return = (
+            [
+                (
+                    "id",
+                    "in",
+                    self.env["stock.quant.package"]
+                    .search([("quant_ids.product_id", "=", self.product_id.id)])
+                    .ids,
+                )
+            ]
+            if self.product_id
```

Comment:
**question**:Would it be possible to avoid the search (line 28) and return

```py
[("quant_ids.product.id", "=", self.product_id.id ])
```
?

Reply:
Updated without search